### PR TITLE
Fixes #1355: Added comments to PR_Template

### LINKED ISSUE #1355: Add comments to PR_TEMPLATE
### Please describe the problem (or idea)

The PR_TEMPLATE Currently has details that could be commented. for example 

> Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

This could be commented. [Here](https://stackoverflow.com/questions/4823468/comments-in-markdown)'s how you add comments in markdown.

### PR DESCRIPTION
**Pull request by KarishmaVanwari**
Fixes #1355   (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below 
* [x] at least 2 reviews required for getting pull request merged
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,8 +12,8 @@ Make sure these boxes are checked before your pull request (PR) is ready to be r
 * [ ] Insert-step functionality is working correct as expected.
 > We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
```
Comment (harshkhandeparkar) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I suggest this should be commented too. Would you like to do it?

Comment (KarishmaVanwari) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done! @harshkhandeparkar

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+<!---Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!--->
 
 * [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
 * [ ] code is in uniquely-named feature branch and has no merge conflicts
 * [ ] PR is descriptively titled
 * [ ] ask `@publiclab/is-reviewers` for help, in a comment below 
 * [ ] at least 2 reviews required for getting pull request merged
 * [ ] Insert-step functionality is working correct as expected.
-> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
+<!--->> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!--->
```
Comment (harshkhandeparkar) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
<!--- We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!--->
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]
:)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (KarishmaVanwari) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done! @harshkhandeparkar

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Yes thank you!!!

**Comment by jywarren** [comment]
Merged!!! 🎉

**Comment by KarishmaVanwari** [comment]
Thanks!!

**Pull request merged**